### ISSUE TITLE
Spelling

### ISSUE THREAD

**Issue by sebastinas** [pwmt/zathura#173]
*On GitLab by @A172 on Mar 28, 2020, 22:17*

---

```
diff --git a/doc/man/zathurarc.5.rst b/doc/man/zathurarc.5.rst
index 2796ff8..de5c4a0 100644
--- a/doc/man/zathurarc.5.rst
+++ b/doc/man/zathurarc.5.rst
@@ -728,8 +728,8 @@ zathura
   value in the list will be used for all other number of pages per row if not set
   explicitly.

-  Per default, the first column is set to 2 for double-page layout, i.e. the faule
-  is set to 1:2. A value of 1:1:3 would put the first page in dual-page layour in
+  Per default, the first column is set to 2 for double-page layout, i.e. the value
+  is set to 1:2. A value of 1:1:3 would put the first page in dual-page layout in
   the first column, and for layouts with more columns the first page would be put
   in the 3rd column.
```

**Comment by sebastinas** [comment]
closed via commit 2d53f77233e261e8aa717fa9c6d0e11e6b9a01cb

**Comment by sebastinas** [comment]
mentioned in commit 2d53f77233e261e8aa717fa9c6d0e11e6b9a01cb
